downloader/build.gradle.kts

```diff
@@ -19,6 +19,7 @@
  */
 
 val jgitVersion: String by project
+val jSchVersion: String by project
```

Comment:
This is not the "jsch" version (and I'd write this in all lower case), but the "jsch-agent-proxy" version. That actually confused me because the latest *JSch* version is [0.1.55](https://mvnrepository.com/artifact/com.jcraft/jsch/0.1.55), but the latest *jsch-agent-proxy* version is [0.0.9](https://mvnrepository.com/artifact/com.jcraft/jsch.agentproxy.jsch/0.0.9) (which depends on *JSch* 0.1.49, so we might want to additionally declare *JSch* in version 0.1.55 to benefit from any [fixes](http://www.jcraft.com/jsch/ChangeLog)).

Reply:
BTW, Gerrit is [discussing to drop JSch](https://bugs.chromium.org/p/gerrit/issues/detail?id=7377) in favor of e.g. Apache Mina, but it seems there is no SSH session factory in JGit for Mia yet.

Reply:
Done. I'll keep 0.0.7 for now because it works and it's the resolved version of the other agent proxy libraries we already have as transitive dependencies through SVN Kit.

Reply:
Does that mean you're worried that SVNKit would not work with version 0.0.9 (which is the version Gradle would align the different *jsch-agent-proxy* versions to)?

Reply:
It's because of urgency. It's already known to work with 0.0.7 and I'd rather merge this and test the upgrade later. Btw, we already use JSch 0.1.55 because of JGit.